pkg/cmd/server/start.go

```diff
@@ -77,11 +80,23 @@ func NewCommandStartAdmissionServer(out, errOut io.Writer, stopCh <-chan struct{
 	}
 
 	flags := cmd.Flags()
-	o.RecommendedOptions.AddFlags(flags)
+	o.AddFlags(flags)
 
 	return cmd
 }
 
+func (o *AdmissionServerOptions) AddFlags(fs *pflag.FlagSet) {
+	o.RecommendedOptions.AddFlags(fs)
+	// first set the UnauthenticatedHTTP2DOSMitigation feature to true by default
```

Comment:
This approach means that the --help from these flags will be listed improperly: it will say false and actually be true.

While probably better than nothing in this case, that is subpar.

Reply:
it is not easy to modify the feature gates

https://github.com/kubernetes/kubernetes/blob/145b84b0b17e5f8e07697f455c6646406bc92149/staging/src/k8s.io/component-base/featuregate/feature_gate.go#L287

So I have modified the flag's help after it is generated. A bit hacky, but it works.